src/PIL/ImageDraw.py

```diff
@@ -181,6 +181,11 @@ def ellipse(self, xy: Coords, fill=None, outline=None, width=1) -> None:
         if ink is not None and ink != fill and width != 0:
             self.draw.draw_ellipse(xy, ink, 0, width)
 
+    def circle(self, xy: Coords, radius, fill=None, outline=None, width=1) -> None:
```

Comment:
```suggestion
    def circle(
        self, xy: Sequence[float], radius: float, fill=None, outline=None, width=1
    ) -> None:
```
`Coords` is either a sequence or a sequence of a sequence. Here, you just want a sequence.
https://github.com/python-pillow/Pillow/blob/bbe1effd63149eb3d43d59efb5fe2659f93700d0/src/PIL/_typing.py#L20